README.md

```diff
@@ -66,18 +66,17 @@ services:
         ports:
             - '3000:3000'
     mesh:
-        image: 0xorg/mesh:0xV3
+        image: 0xorg/mesh:7.2.1-beta-0xv3
         environment:
             ETHEREUM_RPC_URL: 'http://ganache:8545'
-            ETHEREUM_CHAIN_ID: '1337'
-            ETHEREUM_NETWORK_ID: '50'
-            USE_BOOTSTRAP_LIST: 'true'
+            ETHEREUM_CHAIN_ID: 1337
             VERBOSITY: 3
-            PRIVATE_KEY_PATH: ''
-            BLOCK_POLLING_INTERVAL: '5s'
-            P2P_LISTEN_PORT: '60557'
```

Comment:
`P2P_LISTEN_PORT` is separate from `RPC_ADDR`. This configuration is trying to use the same port for both the RPC server and p2p networks, which doesn't work.